website/src/components/markdown/Table.js

```diff
@@ -87,10 +87,15 @@ export const Table = props => {
               '&:not(:first-of-type)': {
                 paddingLeft: gridSize,
               },
+
               '&:not(:last-of-type)': {
                 paddingRight: gridSize,
               },
 
+              '&:last-of-type': {
+                whiteSpace: 'normal',
```

Comment:
This looks a bit weird on small devices. I'll try to find a solution in another PR

Reply:
This is also super arbitrary. But yeah we can pick it up in another PR

Reply:
Yeah, _most of the time_ it'll be a description, but that's not really declarative